containers/scripts/run_hub.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh -ex
+#
+
+if [[ -e /src ]]; then
+    # Inside podman-compose
+    pushd /src
+    cp settings_local.ci.py /usr/lib/python3.9/site-packages/osh/hub/settings_local.py
+    cp osh-hub-httpd.conf /etc/httpd/conf.d/osh-hub-httpd.conf
+    # rm -f /etc/httpd/conf.d/ssl.conf
+    popd
+else
+    # We are in OpenShift deployment. Copy configuration files from persistent storage.
+    # Probably set the up through ansible
+    # curl -o /mnt/osh-configs/osh-hub-httpd.conf https://raw.githubusercontent.com/siteshwar/openscanhub-deployment-configs/main/fedora-infra/osh-hub-httpd.conf 
+    # curl -o /mnt/osh-configs/settings_local.ci.py https://raw.githubusercontent.com/siteshwar/openscanhub-deployment-configs/main/fedora-infra/settings_local.ci.py
+    # cp /mnt/osh-configs/settings_local.ci.py /usr/lib/python3.9/site-packages/osh/hub/settings_local.py
+    # cp /mnt/osh-configs/osh-hub-httpd.conf /etc/httpd/conf.d/osh-hub-httpd.conf
+    echo "Do nothing here!"
+    # cp /etc/keytabs/service.keytab
+fi
+
+for _ in $(seq 100); do
+    # TODO: Put hostname of the db01 server here
+    pg_isready -h fedora-osh-db && break
```

Comment:
This should be configurable and kept separately for staging and production deployments. Or we can assume that central database is always up?